docs/release-notes/breaking-changes.md

```diff
@@ -12,6 +12,114 @@ If you are migrating from a version prior to version 9.0, you must first upgrade
+Queries that include parentheses in unquoted index names will now result in a parsing exception.
+
+**Action:**
+Update affected queries to quote index names that contain parentheses. For example, use `FROM "("foo")"` instead of `FROM (foo)`.
+For more information, view [#130427](https://github.com/elastic/elasticsearch/pull/130427) (issue: [#130378](https://github.com/elastic/elasticsearch/issues/130378))
+:::
+
+:::{dropdown} Disallows mixing quoted and unquoted components in `FROM` index patterns
+ES|QL no longer allows mixing quoted and unquoted parts in `FROM` index patterns (e.g. `FROM remote:"index"`). Previously, such patterns were parsed inconsistently and could result in misleading runtime errors.
+
+**Impact:**
+Queries using partially quoted index patterns—such as quoting only the index or only the remote cluster—will now be rejected at parse time. This change simplifies grammar handling and avoids confusing validation failures.
+
+**Action:**
+Ensure index patterns are either fully quoted or fully unquoted. For example:
+* Valid: `FROM "remote:index"` or `FROM remote:index`
+* Invalid: `FROM remote:"index"`, `FROM "remote":index`
+
+For more information, view [#127636](https://github.com/elastic/elasticsearch/pull/127636) (issue: [#122651](https://github.com/elastic/elasticsearch/issues/122651))
+:::
+
+:::{dropdown} `skip_unavailable` now catches all remote cluster runtime errors in ES|QL
+When `skip_unavailable` is set to `true`, ES|QL now treats all runtime errors from that cluster as non-fatal. Previously, this setting only applied to connectivity issues (i.e. when a cluster was unavailable).
+
+**Impact:**
+Errors such as missing indices on a remote cluster will no longer cause the query to fail. Instead, the cluster will appear in the response metadata as `skipped` or `partial`.
+
+**Action:**
+If your workflows rely on detecting remote cluster errors, review your use of `skip_unavailable` and adjust error handling as needed.
+
+For more information, view [#128163](https://github.com/elastic/elasticsearch/pull/128163)
+:::
+
+Snapshot/Restore:
+
+:::{dropdown} Upgrades `repository-s3` plugin to AWS SDK v2
+The `repository-s3` plugin now uses AWS SDK v2 instead of v1, as AWS will deprecate SDK v1 before the end of Elasticsearch 8.19’s support period. The two SDKs differ in behavior, which may require updates to your configuration.
+
+**Impact:**
+Existing `repository-s3` configurations may no longer be compatible. Notable differences in AWS SDK v2 include but not limited to:
+- AWS SDK v2 requires users to specify the region to use for signing requests, or else to run in an environment in which it can determine the correct region automatically. The older SDK would try to determine the region based on the endpoint URL as specified with the `s3.client.${CLIENT_NAME}.endpoint` setting, together with other data drawn from the operating environment, and would ultimately fall back to `us-east-1` if no better value could be found.
```

Comment:
```suggestion
- AWS SDK v2 requires users to specify the region to use for signing requests, or else to run in an environment in which it can determine the correct region automatically. The older SDK used to determine the region based on the endpoint URL as specified with the `s3.client.${CLIENT_NAME}.endpoint` setting, together with other data drawn from the operating environment, and fell back to `us-east-1` if no better value was found.
```

I think switching from conditionals to past tense makes more sense when describing old behaviours vs new.